yaml/pois.yaml

```diff
@@ -1119,17 +1119,17 @@ filters:
       <<: *output_properties
       kind: mall
   - filter: {amenity: prison}
-    min_zoom: { clamp: { min: 0, max: 15, value: { sum: [ { col: zoom }, 2.55 ] } } }
+    min_zoom: { clamp: { min: 12, max: 15, value: { sum: [ { col: zoom }, 2.55 ] } } }
```

Comment:
Should be min: `13`, please.

Reply:
- `prison` is important, but shouldn't be so prominent – I think zoom 13 would be good, and same zoom we limit hospitals to (which are often bigger). Here's SF JV hall:
    ![image](https://user-images.githubusercontent.com/853051/50493445-6b549c00-09d2-11e9-933a-5afe1d97a7e9.png)
    and the 1st prison in California is only good for zoom 13: https://www.openstreetmap.org/way/304320226
    same for New York City's finest at zoom 13: https://www.openstreetmap.org/relation/3955540